### PR TITLE
Adding `ocdev url` tests

### PR DESCRIPTION
**Pull request by ashetty1**

### CODE REVIEW

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -188,31 +188,36 @@ var _ = Describe("Usecase #5", func() {
 			// Push the changes
 			runCmd("ocdev push --local " + tmpDir + "/nodejs-ex")
 
-			// ping the ip
-			pingSvc(pingUrl)
+			// Create route
+			runCmd("ocdev url create nodejs")
+			getRoute := runCmd("ocdev url list  | sed -n '1!p' | awk '{ print $3 }'")
+			getRoute = strings.TrimSpace(getRoute)
+			Ω(getRoute).Should(ContainSubstring("nodejs-" + projName))
```
Comment (kadel) [inline tests/e2e/e2e_test.go]: what is Ω ?

Comment (ashetty1) [reply tests/e2e/e2e_test.go]: @kadel ginkgo/geomega notation

Comment (surajnarwade) [reply tests/e2e/e2e_test.go]: @ashetty1 , how to type that (/me curious) ?

Comment (kadel) [reply tests/e2e/e2e_test.go]: it looks really strange. Is there an alternative  to using  Ω? Something that is a little bit more standard?

**Pull request merged**